### ISSUE TITLE
Doubled messages on Heroku

### ISSUE THREAD

**Issue by baob** [baob/submit-this#3]
Problem with an events API based app running on heroku (free tier). Seems that something,  perhaps the heroku infrastructure, perhaps slack itself, randomly sends 1-2 extra copies of ‘app_mention’ events.

Maybe slack saw timeouts or sluggish response from heroku and repeated the events.

**Comment by baob** [comment]
Help from "Bot developer hangout
> This sounds like your lambda is not responding back to Slack with a HTTP 200  so the Slack Events API does not know that you actually received the message and so we re-resend the event. Check out https://api.slack.com/events-api#responding_to_events and take note of "Failure conditions" and "Graceful retries" for more info on how to debug if and why we are retrying

**Comment by baob** [comment]
Might have been fixed by commit 5a4db41 but until we see the retries coming through on the logs, and the app suppressing them, then we won't know for sure.

**Comment by baob** [comment]
Just seen the suppression working:

```
2020-04-18T12:03:57.304717+00:00 app[web.1]: suppressing retry based on headers: {
2020-04-18T12:03:57.304719+00:00 app[web.1]: 'user-agent': 'Slackbot 1.0 (+https://api.slack.com/robots)',
2020-04-18T12:03:57.304719+00:00 app[web.1]: accept: '*/*',
2020-04-18T12:03:57.304720+00:00 app[web.1]: 'accept-encoding': 'gzip,deflate',
2020-04-18T12:03:57.304721+00:00 app[web.1]: 'content-type': 'application/json',
2020-04-18T12:03:57.304721+00:00 app[web.1]: 'x-slack-retry-num': '2',
2020-04-18T12:03:57.304721+00:00 app[web.1]: 'x-slack-retry-reason': 'http_timeout',
2020-04-18T12:03:57.304723+00:00 app[web.1]: 'x-forwarded-for': '3.95.2.106',
2020-04-18T12:03:57.304724+00:00 app[web.1]: 'x-forwarded-proto': 'https',
2020-04-18T12:03:57.304724+00:00 app[web.1]: 'x-forwarded-port': '443',
2020-04-18T12:03:57.304724+00:00 app[web.1]: via: '1.1 vegur',
2020-04-18T12:03:57.304725+00:00 app[web.1]: 'connect-time': '0',
2020-04-18T12:03:57.304725+00:00 app[web.1]: 'x-request-start': '1587211437301',
2020-04-18T12:03:57.304725+00:00 app[web.1]: 'total-route-time': '0',
2020-04-18T12:03:57.304726+00:00 app[web.1]: 'content-length': '696'
2020-04-18T12:03:57.304726+00:00 app[web.1]: }|
```
